grouper/fe/templates/help.html

```diff
@@ -17,8 +18,95 @@
+of C.) That's all there is to it.  Technically speaking, the users and groups constitute
+the nodes of a <a href="http://en.wikipedia.org/wiki/Directed_acyclic_graph">directed
+acyclic graph</a> and a user or group is a member of any of its ancestors.</p>
+
+<h3>What's the difference between the member, manager, and owner roles?</h3>
+<p>These are the three types of group membership for users: a member just inherits the
+group's permissions, a manager is allowed to process requests to join a group, and an
+owner has full control over the group (including the ability to change the membership type
+of other users or add/remove the group's permissions.)  Technically, groups are members of
+other groups.</p>
+
+<h3>What's the deal with permissions?</h3>
+<p>Grouper <a href="/permissions">permissions</a> are just enumerated strings. First,
+permissions are first created (added to the set of available permissions.) Then
+permissions are granted to groups as (permission, argument) pairs where an argument is an
+optional second string. Finally, users inherit the union of all (permission, argument)
+pairs from the groups of which they are members.</p>
+
+<h3>Who can create groups?</h3>
+<p>Everyone.  The user creating a group automatically becomes an owner of that group,
+which begins with no permissions.</p>
+
+<h3>Who can create users?</h3>
+<!-- TODO: update language once users can be prepopulated. -->
+<p>Users are automatically populated into Grouper as they visit the web UI.  We're working
+on a manual way to pre-populate users (see user_admin below).</p>
+
+<h3>Who can create permissions?</h3>
+<p>A user with the permission ({{permission(create_perm)}}, prefix.*) can create any
+permission starting with "prefix." We strongly encourage namespacing using the character
+'.' as a separator.</p>
+
+<h3>Who can grant permissions?</h3>
+<p>Generally, a user with the permission ({{permission(grant_perm)}}, prefix.*) can grant
+any permission starting with "prefix." with any argument.  As a special case, the
+permission ({{grant_perm.name}}, {{grant_perm.name}}/prefix.*) allows a user to
+grant the permission to grant ({{grant_perm.name}}, prefix.*).</p>
+
+<h3>How does auditing work?</h3>
+<!-- FIXME(mildorf): does the argument matter for grouper.auditor.permission? -->
+<p>Grouper permissions have a bit that defines whether they are audited or not.  Groups
+with at least one audited permission become audited groups.  Users with the
+{{permission(audit_perm)}} permission (any argument) are considered auditors.  Only
+auditors can be the manager or owner of an audited group.</p>
+
+<!-- TODO: change once auditing tooling is ready! -->
+<p>We're working on tooling to facilitate regular monitoring of both members of audited
+groups and groups with audited permissions.</p>
+
+<h3>How do I set up Grouper?</h3>
+<p>See the README that comes with Grouper. Basically, after a pip install, you need to
```

Comment:
Same comment here. We shouldn't have setup information on the help page. This is only viewable after you have the site set up.